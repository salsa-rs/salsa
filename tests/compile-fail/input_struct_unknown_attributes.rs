#[salsa::input]
struct InputWithUnknownAttrs {
    /// Doc comment
    field: u32,
    #[anything]
    field2: u32,
}

fn main() {}
