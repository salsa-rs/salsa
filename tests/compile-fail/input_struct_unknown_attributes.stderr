error: cannot find attribute `anything` in this scope
 --> tests/compile-fail/input_struct_unknown_attributes.rs:5:7
  |
5 |     #[anything]
  |       ^^^^^^^^
